server/controllers/entities/lib/has_works_labels_occurrence.coffee

```diff
@@ -69,4 +67,13 @@ hasBnfOccurrence = (authorEntity, worksLabels)->
   # is wrong and we can't know which
   if bnfIds?.length isnt 1 then return false
   getBnfAuthorWorksTitles bnfIds[0]
-  .then (titles)-> _.haveAMatch titles, worksLabels
+  .then createOccurencDocs worksLabelsPattern, authorEntity
+
+createOccurenceDocs = (worksLabelsPattern, authorEntity)->
+  return (article)->
+    unless article.extract.match(worksLabelsPattern)?
+      return []
+    return
+      uri: authorEntity.uri
+      url: article.url
+      patternMatch: worksLabelsPattern
```

Comment:
regex can't be saved as such as they aren't part of the JSON standard:
```coffee
JSON.stringify { foo: /bar/ }
# => '{"foo":{}}'
```
But anyway, storing the full `worksLabelsPattern` isn't that useful, the really interesting info would be to know which title matched